docs/content/docs/connectors/table/jdbc.md

```diff
@@ -308,47 +308,51 @@ As there is no standard syntax for upsert, the following table describes the dat
     </tbody>
 </table>
 
-### Postgres Database as a Catalog
+JDBC Catalog
+------------
 
 The `JdbcCatalog` enables users to connect Flink to relational databases over JDBC protocol.
 
-Currently, `PostgresCatalog` is the only implementation of JDBC Catalog at the moment, `PostgresCatalog` only supports limited `Catalog` methods include:
+Currently, `PostgresCatalog` and `MySQLCatalog` are the two implementations of JDBC Catalog at the moment, `JdbcCatalog` only supports limited `Catalog` methods include:
 
 ```java
-// The supported methods by Postgres Catalog.
-PostgresCatalog.databaseExists(String databaseName)
-PostgresCatalog.listDatabases()
-PostgresCatalog.getDatabase(String databaseName)
-PostgresCatalog.listTables(String databaseName)
-PostgresCatalog.getTable(ObjectPath tablePath)
-PostgresCatalog.tableExists(ObjectPath tablePath)
+// The supported methods by Postgres & MySQL Catalog.
+databaseExists(String databaseName);
+listDatabases();
+getDatabase(String databaseName);
+listTables(String databaseName);
+getTable(ObjectPath tablePath);
+tableExists(ObjectPath tablePath);
 ```
 
 Other `Catalog` methods is unsupported now.
 
-#### Usage of PostgresCatalog
+### Usage of JDBC Catalog
 
-Please refer to [Dependencies](#dependencies) section for how to setup a JDBC connector and Postgres driver.
+The section mainly describes how to create and use a `PostgresCatalog` or `MySQLCatalog`.
+Please refer to [Dependencies](#dependencies) section for how to setup a JDBC connector and Postgres/MySQL driver.
```

Comment:
It would be good to avoid explicitly listing all supported implementations where possible, e.g.

```suggestion
Please refer to [Dependencies](#dependencies) section for how to setup a JDBC connector and the corresponding driver.
```